src/scripts/main.js

```diff
@@ -1,3 +1,38 @@
 'use strict';
 
-// write code here
+
+const list = document.querySelector('ul');
+const people = document.querySelectorAll('li');
+
+
+function sortList(arr) {
+  function transNum(num) {
+    return Number(num.split('$').join('').split(',').join(''));
+  }
+
+  return arr.map(item => {
+    const dataInfo = item.dataset;
+    const innerText = `${item.innerText}`;
+
+    return {name : innerText, ...dataInfo}
```

Comment:
don't forget about code style
```suggestion
    return { ...dataInfo, name: innerText }
```